reference-architecture/osp-cli/ch5.8.5_configure_docker_storage_ansible.sh

```diff
@@ -3,6 +3,6 @@ cat <<EOF >docker-storage-setup
 DEVS=/dev/vdb
 VG=docker-vg
 EOF
-ansible nodes -i inventory -m copy -a "src=docker-storage-setup dest=/etc/sysconfig/docker-storage-setup force=yes"
+ansible app-nodes,infra-nodes -i inventory -m copy -a "src=docker-storage-setup dest=/etc/sysconfig/docker-storage-setup force=yes"
 ansible nodes -i inventory -a "/usr/bin/docker-storage-setup"
```

Comment:
Masters will end up with a loopback file for docker images but as they are not going to run containers, it's ok.